landscape.yml

```diff
@@ -15849,6 +15844,12 @@ landscape:
             logo: cardinal-health.svg
             crunchbase: https://www.crunchbase.com/organization/cardinal-health
             joined: '2021-08-27'
+          - item:
+            name: CARIAD (supporter)
+            homepage_url: https://www.cariad.us
+            logo: cariad.svg 
+            crunchbase: https://www.crunchbase.com/organization/cariad
+            joined: '2023-08-24’             
```

Comment:
Hi @Swil78,

The `landscape.yml` file validation has failed because the `joined` field isn't using a straight single quote at the end.

Hope this helps 🙂